src/lib/VirtualScroller.js

```diff
@@ -254,7 +254,7 @@ class VirtualScroller {
         }
 
         // Create a new list element to be swapped out for the existing one
-        const newListEl = this.createListElement();
+        const newListEl = document.createDocumentFragment();
```

Comment:
The naming here seems a little confusing, since later we append this to `this.listEl`.